floodviz/views.py

```diff
@@ -57,8 +67,10 @@ def _peakflow_helper():
     peak_data = peak_flow_utils.parse_peak_data(historic_peaks, daily_value_data)
     return peak_data
 
+
 def _map_helper():
     site_data = map_utils.site_dict(ref['site_ids'], app.config['NWIS_SITE_SERVICE_ENDPOINT'])
+    linked_data.set_gages(site_data)
```

Comment:
After seeing this, I think you made the right decision regarding my previous comment about setters vs passing as object params.